content/en/docs/concepts/storage/volumes.md

```diff
@@ -1168,7 +1168,9 @@ persistent volume:
   secrets are passed.  When you have configured secret data for node-initiated
   volume expansion, the kubelet passes that data via the `NodeExpandVolume()`
   call to the CSI driver. In order to use the `nodeExpandSecretRef` field, your
-  cluster should be running Kubernetes version 1.25 or later and you must enable
+  cluster should be running Kubernetes version 1.27 or later. This feature has
```

Comment:
why bump the version here? in 1.25 it was usable, although the feature was alpha stage.